src/test/scala/chiselTests/BundleEltsSpec.scala

```diff
@@ -0,0 +1,84 @@
+// See LICENSE for license details.
```

Comment:
I'm not sure where this came from, probably git add'd something from another branch. Gone now.